utils/firestore.js

```diff
@@ -999,13 +999,14 @@ const reportMissingSpecimen = async (siteAcronym, requestData) => {
 
 }
 
-const searchSpecimen = async (masterSpecimenId, siteCode) => {
+const searchSpecimen = async (masterSpecimenId, siteCode, allSitesFlag) => {
     const snapshot = await db.collection('biospecimen').where('820476880', '==', masterSpecimenId).get();
-    if(snapshot.size === 1) {
+    if (snapshot.size === 1) {
         const token = snapshot.docs[0].data().token;
```

Comment:
If all site search is enabled, further check in `participants` is not necessary, right?
Correct me if I'm mistaken. Below line 1004, the result can be returned, like so:
```js
if (allSitesFlag) return snapshot.docs[0].data();
```